Tasks/PublishToAzureServiceBusV1/task.json

```diff
@@ -52,14 +52,23 @@
                 "editorExtension": "ms.vss-services-azure.azure-servicebus-message-grid"
             }
         },
+		{
+			"name": "sessionId",
+			"type": "string",
+			"label": "Session Id for Message",
+			"required": false,
+			"defaultValue": "",
+			"helpMarkDown": "This is the session id whith which you want to post the message. In case not given , default Guid will be assigned to it. Used for Session Based Queues, for Non Session Based Queues, it would not matter.",
```

Comment:
Please fix up the help markdown as we are no longer generating default Guid

Reply:
Done